pgxn/neon/neon--1.0--1.1.sql

```diff
@@ -0,0 +1,10 @@
+\echo Use "ALTER EXTENSION neon UPDATE TO '1.1'" to load this file. \quit
+
+CREATE FUNCTION neon_get_stats()
+RETURNS SETOF RECORD
+AS 'MODULE_PATHNAME', 'neon_get_stats'
+LANGUAGE C PARALLEL SAFE;
+
+-- Create a view for convenient access.
+CREATE VIEW neon_stats AS
```

Comment:
I suggest to be more specific and rename this to `neon_lfc_stats` and `neon_get_lfc_stats()` respectively, wdyt?

Reply:
My original idea was that this view can be used not only forLFC but for any other Neon specific activities, for example prefetching. But maybe having specialised views is better - not sure here.